packages/lab/src/tokenized-input-next/TokenizedInputNext.tsx

```diff
@@ -0,0 +1,440 @@
+import {
+  Button,
+  ButtonProps,
+  makePrefixer,
+  NecessityType,
+  StatusAdornment,
+  useDensity,
+  useForkRef,
+  useId,
+  useIsomorphicLayoutEffect,
+} from "@salt-ds/core";
+import {
+  ChangeEventHandler,
+  FocusEvent,
+  FocusEventHandler,
+  ForwardedRef,
+  forwardRef,
+  HTMLAttributes,
+  KeyboardEvent,
+  KeyboardEventHandler,
+  ReactEventHandler,
+  Ref,
+  SyntheticEvent,
+  TextareaHTMLAttributes,
+  useCallback,
+  useRef,
+  useState,
+} from "react";
+import {
+  TokenizedInputNextHelpers,
+  TokenizedInputNextState,
+  useTokenizedInputNext,
+} from "./useTokenizedInputNext";
+import { clsx } from "clsx";
+import { InputPill } from "./internal/InputPill";
+import { CloseIcon, OverflowMenuIcon } from "@salt-ds/icons";
+import { useWidth } from "./internal/useWidth";
+import { useResizeObserver } from "./internal/useResizeObserver";
+import { calcFirstHiddenIndex } from "./internal/calcFirstHiddenIndex";
+import { useWindow } from "@salt-ds/window";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import tokenizedInputCss from "./TokenizedInputNext.css";
+
+type ChangeHandler<Item> = (selectedItems: Item[] | undefined) => void;
+
+type ExpandButtonProps = Pick<
+  ButtonProps,
+  "role" | "aria-roledescription" | "aria-describedby"
+> & { accessibleText?: string };
+
+export interface TokenizedInputNextProps<Item>
+  extends Partial<TokenizedInputNextState<Item>>,
+    Omit<
+      HTMLAttributes<HTMLDivElement>,
+      "onFocus" | "onBlur" | "onChange" | "onKeyUp" | "onKeyDown"
+    > {
+  ExpandButtonProps?: ExpandButtonProps;
+  disabled?: boolean;
+  focused?: boolean;
+  expandButtonRef?: Ref<HTMLButtonElement>;
+  onFocus?: FocusEventHandler<HTMLTextAreaElement | HTMLButtonElement>;
+  onBlur?: FocusEventHandler<HTMLTextAreaElement | HTMLButtonElement>;
+  onKeyUp?: KeyboardEventHandler<HTMLTextAreaElement | HTMLButtonElement>;
+  // Can key down on either input or expand button
+  onKeyDown?: KeyboardEventHandler<HTMLTextAreaElement | HTMLButtonElement>;
+  onRemoveItem?: (index: number) => void;
+  onInputBlur?: FocusEventHandler<HTMLTextAreaElement>;
+  onInputFocus?: FocusEventHandler<HTMLTextAreaElement>;
+  onInputChange?: ChangeEventHandler<HTMLTextAreaElement>;
+  onInputSelect?: ReactEventHandler<HTMLTextAreaElement>;
+  onClick?: (event: SyntheticEvent<HTMLElement>) => void;
+  onClear?: ReactEventHandler;
+
+  delimiter?: string | string[];
```

Comment:
Is it simpler to just have a prop called `delimiters` and only to take an array?